inline_verifier.go

```diff
@@ -275,6 +279,10 @@ func (v *InlineVerifier) Wait() {
 	v.backgroundVerificationWg.Wait()
 }
 
+func (v *InlineVerifier) Message() string {
+	return fmt.Sprintf("BinlogVerifyStore.currentRowCount = %d", v.reverifyStore.CurrentRowCount())
```

Comment:
Does it matter this is not _Binlog**Re**verifyStore_?

Reply:
The actual data structure is called BinlogVerifyStore for the InlineVerifier. We keep calling it the reverify store because that's what the IterativeVerifier does, but it's not as accurate of a name given that the InlineVerifier keeps that store small (where as the IterativeVerifier did not and let it grow basically unbounded).